src/wallet.js

```diff
@@ -0,0 +1,90 @@
+import { existsSync, readFileSync, writeFileSync } from 'fs'; // Import the fs module for file operations
+import { GetMempool } from './Mempool';
+import { Transaction, UnsignedTransaction } from './Transaction';
+
+
+class Wallet {
+    constructor() {
+        if (existsSync('private_key.json')) {
+            const { PrivateKey, Password } = this.loadFromFile();
+            this.PrivateKey = PrivateKey;
+            this.Password = Password;
+        } else {
+            this.Password = this.generatePassword();
+            this.PrivateKey = this.generatePrivatePemString(this.password);
+            this.saveToFile();
+        }
+        this.PublicKey = this.generatePublicPemString(this.PrivateKey, this.Password);
+    }
+
+    sendMoney(ReceiverPks, Msgs) {
+        let MoneyToSend = 0;
+        for (const m of Msgs) {
+            MoneyToSend += m;
+        }
+        const Tx = this.createTransaction(this.getUTXOs(MoneyToSend), ReceiverPks, Msgs);
+        this.insertToMempool(Tx);
+    }
+
+    getUTXOs(Money) {
+        const Blockchain = getBlockchain();
+        const utxos = Blockchain.getUTXOs(this.generatePublicPemString(this.PrivateKey, this.Password));
+
+        if (Array.isArray(utxos)) {
+            console.log("UTXOs are inside list")
+        } else {
+            console.log("UTXOs are not a list")
+        }
+
+        const ListOfValidUTXO = utxos.filter(i => Blockchain.isValidUTXO(i));

```

Comment:
`ListOfValidUTXO` constant is unused

Reply:
Also you attempt to filter valid UTXOs but don't handle the case when there are not enough UTXOs to cover the requested amount. You should check if `TotalAmount` is less than `Money` after the loop and handle this situation